lib/core/util.js

```diff
@@ -895,6 +894,13 @@ function getCodings (contentEncoding) {
   return codings
 }
 
+let createInflate
+
+function lazyCreateInflate (zlibOptions) {
+  createInflate ??= require('../web/fetch/util').createInflate
```

Comment:
@KhafraDev, wouldn't it make more sense to lift `createInflate` to `../core/util` in this case? If multiple modules depend on it, perhaps it's a sign to make it a generic utility.

Reply:
I went for the path of least resistance, I have no objections in either case.